strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/ConfigurationManagementServiceImpl.java

```diff
@@ -618,4 +624,57 @@ private void modifyInLock(final Consumer<MutableConfiguration> operation,
         }
     }
 
+    @Override
+    public boolean setMaxUploadFileSize(String maxUploadSizeString) throws SizeFormatValidationException
+    {
+        if (!(this.multipartConfigElement instanceof UpdatableMultipartConfigElement))
+        {
+            return false;
+        }
+
+        maxUploadSizeString.replaceAll("\\s+","");
+        maxUploadSizeString = maxUploadSizeString.toUpperCase();
+
+        Long uploadSize;
+        if(maxUploadSizeString.equals("UNLIMITED"))
```

Comment:
Cosmetic: Please, new space after the `if`.

Reply:
done